doc/interop-test-descriptions.md

```diff
@@ -1143,10 +1147,15 @@ Procedures:
       }
     }
     ```
-The call carries a reference to receive the load report, e.g. using CallOptions.
-The reference will be passed to the custom LB policy as part of the `OrcaOobReportListener` API.
-* Client asserts that, after 1.5 second, the latest OOB load report received is equal to the test load report.
-* Client sends another unary call to the server. The call request sets `orca_oob_report` to a 
+The test then blocks until the first response is received. This means the server lock is acquired.
+It is important that if the stream is completed or terminated with error at this time, 
+the test aborts and fails.
+* Client samples the latest OOB load report by doing empty unary calls. The call 
```

Comment:
Instead of having to make additional unary calls to do this, how about simply having the LB policy invoke an injected callback that the client provides at the time that it registers the LB policy?  This is essentially what we do for C++ in our end-to-end tests:

https://github.com/grpc/grpc/blob/d9f64437b09d97bd9aebc6bfa46d5a61e7477c41/test/cpp/end2end/client_lb_end2end_test.cc#L2579

This way, there's no polling needed; instead, we get a callback when each OOB backend metric report is received.

Reply:
Yea no polling has its advantage. It avoids sleeps between each polling, and processes immediately after callback is invoked. 
Part of the reason java does polling is because it saves some complexity of extra injection, both implementation and description here. In java can re-use what we already have (using callOptions)for per-query test so it would be easier to reason and consistent.
In my opinion either way would work, what about I make the doc less specific and allow each language to implement their verification.  The timeout should be the same tho.